src/main/java/com/verily/genomewarp/utils/GvcfToVcfAndBed.java

```diff
@@ -19,92 +19,145 @@
 import java.io.IOException;
 import java.io.PrintWriter;
 import java.nio.file.Files;
+import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
 /**
- * Helper class for extracting a canonical VCF and a BED file from a gVCF file
+ * Helper class for extracting a canonical VCF and a BED file from a gVCF file.
  */
 public class GvcfToVcfAndBed {
-
-  private static final Logger logger = Logger.getLogger(VcfToVariant.class.getName());
-
+  // class constants
+  private static final Logger logger = Logger.getLogger(GvcfToVcfAndBed.class.getName());
+  
+  private static final int MIN_VCF_FIELDS = 8;
+  private static final int START_FIELD_INDEX = 1;
+  private static final int END_FIELD_INDEX = 3;
+  private static final int ALT_BASES_FIELD_INDEX = 4;
+  private static final int FILTER_FIELD_INDEX = 6;
+  private static final int FORMAT_FIELD_INDEX = 7;
+  private static final String UNKNOWN_VALUE = ".";
+  private static final String GVCF_ALT_ALLELE = "<*>";
+  private static final String HEADER_START_STR = "#";
+  private static final String FIELD_SEP_REGEX = "\\t";
+  private static final String PASS_FILTER_VALUE = "PASS";
+  private static final String FORMAT_FIELD_SEP_REGEX = ";";
+  private static final String FORMAT_END_KEY = "END";
+  private static final int CHROM_FIELD_INDEX = 0;
```

Comment:
nit: put this between current lines 34-35 so the field indices all appear together and in order.